### ISSUE TITLE
[NOT AN ISSUE] Does Unity Catalog supports hive-style partitioning?

### ISSUE THREAD

**Issue by SemyonSinchenko** [unitycatalog/unitycatalog#106]
**Is your feature request related to a problem? Please describe.**

In the [TableInfo](https://github.com/unitycatalog/unitycatalog/blob/main/api/Models/TableInfo.md) model there is no information about partition column. Does UC actually supports partitioning? Is it possible to create a partitioned table via an API?

**Describe the solution you would like**

A separate field in `TableInfo` that contains information about partition columns. This field may be empty, of course.

**Additional context**

.

**Comment by tdas** [comment]
Hey @SemyonSinchenko, the [column info](https://github.com/unitycatalog/unitycatalog/blob/main/api/Models/ColumnInfo.md) contains optional partition index.

**Comment by SemyonSinchenko** [comment]
> Hey @SemyonSinchenko, the [column info](https://github.com/unitycatalog/unitycatalog/blob/main/api/Models/ColumnInfo.md) contains optional partition index.

How is the partition index of columns corresponding to the partitioning of the table?

When I'm registering a delta table, for example, in Hive metastore, I can specify the partition column:

```sql
CREATE TABLE my_table (
)
USING delta
PARTITIONED BY my_partition_column
```

Is it possible to register a partitioned table in UC?

**Comment by tdas** [comment]
Yes, it is possible. partitioning index is ordered in the same way the order of partition columns specified the CREATE TABLE.
